resources/game_data/resource_generator/resource_generator.gd

```diff
@@ -14,10 +14,16 @@ class_name ResourceGenerator
 @export var hidden: bool = false
 @export var max_amount: int = -1
 @export var column: int = 0
+@export var dynamic_efficiency_id: String
 
 var _random_drops_sum: int = -1
 
 
+## if idle production of this resource can be increasing, e.g. (mason -> house ->) worker -> food
```

Comment:
Currently, "food" is the only such resource (is produced faster every cycle, i.e. houses produce peasants which produce food).